packages/web-components/src/components/va-notification/va-notification.tsx

```diff
@@ -0,0 +1,126 @@
+import { 
+  Component, 
+  Element,
+  Event,
+  EventEmitter,
+  Host, 
+  Prop, 
+  h 
+} from '@stencil/core';
+import classnames from 'classnames';
+
+/**
+ * @componentName Notification
+ * @maturityCategory dont_use
+ * @maturityLevel proposed
+ */
+
+@Component({
+  tag: 'va-notification',
+  styleUrl: 'va-notification.scss',
+  shadow: true,
+})
+export class VaNotification {
+  @Element() el!: any;
+
+  /**
+   * If `true`, the card will be visible.
+   */
+  @Prop() visible?: boolean = true;
+
+  /**
+   * Symbol indicates type of notification 
+   * Current options are: action-required, update
+   */
+  @Prop() symbol?: string = 'none';
+
+  /**
+   * Aria-label text for the close button.
+   */
+  @Prop() closeBtnAriaLabel?: string = 'Close notification';
+
+  /**
+   * If `true`, a close button will be displayed.
+   */
+  @Prop({ reflect: true }) closeable?: boolean = false;
```

Comment:
Do we want to make this button functional for the actual visibility of the component? I'm thinking similar to how we have a `visible` prop in [va-banner](https://design.va.gov/storybook/?path=/docs/components-va-banner--dismissible) and [va-modal](https://design.va.gov/storybook/?path=/docs/components-va-modal--default).